railties/test/commands/middleware_test.rb

```diff
@@ -196,10 +196,10 @@ def app
     assert_includes middleware, "ActionDispatch::AssumeSSL"
   end
 
-  test "ActionDispatch::SSL is present when force_ssl is set" do
```

Comment:
@dhh did you mean to delete this test?

Reply:
I did not. Not sure what happened there. Please do PR to restore!

Reply:
https://github.com/rails/rails/pull/52812